[2024-10-08T19:52:05.094+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-08T19:52:05.117+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: update_database.get_starting_date manual__2024-10-08T19:52:02.822048+00:00 [queued]>
[2024-10-08T19:52:05.130+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: update_database.get_starting_date manual__2024-10-08T19:52:02.822048+00:00 [queued]>
[2024-10-08T19:52:05.131+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-10-08T19:52:05.150+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): get_starting_date> on 2024-10-08 19:52:02.822048+00:00
[2024-10-08T19:52:05.157+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=4423) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-08T19:52:05.160+0000] {standard_task_runner.py:63} INFO - Started process 4425 to run task
[2024-10-08T19:52:05.160+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'update_database', 'get_starting_date', 'manual__2024-10-08T19:52:02.822048+00:00', '--job-id', '3855', '--raw', '--subdir', 'DAGS_FOLDER/update.py', '--cfg-path', '/tmp/tmpkb1oj4li']
[2024-10-08T19:52:05.162+0000] {standard_task_runner.py:91} INFO - Job 3855: Subtask get_starting_date
[2024-10-08T19:52:05.204+0000] {task_command.py:426} INFO - Running <TaskInstance: update_database.get_starting_date manual__2024-10-08T19:52:02.822048+00:00 [running]> on host 1fee8bc549a8
[2024-10-08T19:52:05.287+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='update_database' AIRFLOW_CTX_TASK_ID='get_starting_date' AIRFLOW_CTX_EXECUTION_DATE='2024-10-08T19:52:02.822048+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-08T19:52:02.822048+00:00'
[2024-10-08T19:52:05.288+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-08T19:52:05.298+0000] {logging_mixin.py:188} INFO - 2024-05-19 15:00:00 Ross County
[2024-10-08T19:52:05.298+0000] {python.py:237} INFO - Done. Returned value was: ('2024-05-19', '15:00:00', 'Ross County')
[2024-10-08T19:52:05.300+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-08T19:52:05.324+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=update_database, task_id=get_starting_date, run_id=manual__2024-10-08T19:52:02.822048+00:00, execution_date=20241008T195202, start_date=20241008T195205, end_date=20241008T195205
[2024-10-08T19:52:05.376+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-08T19:52:05.405+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-08T19:52:05.407+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
